Plugins/Skinning/src/Display/SkinningDisplayComponent.hpp

```diff
@@ -139,7 +139,7 @@ class SKIN_PLUGIN_API SkinningDisplayComponent : public Ra::Engine::Component {
             std::string name = m_name + "_Partition";
 
             Ra::Engine::RenderObject* renderObject =
-                new Ra::Engine::RenderObject( name, this, Ra::Engine::RenderObjectType::Fancy );
+                new Ra::Engine::RenderObject( name, this, Ra::Engine::RenderObjectType::Geometry );
```

Comment:
There is an issue with the rendering of cameras due to the rendering architecture.
Please Restore to  `Geometry`.